README.md

```diff
@@ -57,6 +57,8 @@ The list below indicates all algorithms supported by liboqs, but not all those a
 - **SPHINCS+-SHA2**: SPHINCS+-SHA2-128f-simple, SPHINCS+-SHA2-128s-simple, SPHINCS+-SHA2-192f-simple, SPHINCS+-SHA2-192s-simple, SPHINCS+-SHA2-256f-simple, SPHINCS+-SHA2-256s-simple
 - **SPHINCS+-SHAKE**: SPHINCS+-SHAKE-128f-simple, SPHINCS+-SHAKE-128s-simple, SPHINCS+-SHAKE-192f-simple, SPHINCS+-SHAKE-192s-simple, SPHINCS+-SHAKE-256f-simple, SPHINCS+-SHAKE-256s-simple
 <!--- OQS_TEMPLATE_FRAGMENT_LIST_SIGS_END -->
```

Comment:
Stuff that's between the template START/END tags will be wiped out by the generation code. If the XMSS and LMS lines were manually generated, they should be after the END tag; if they're automatically generated, then they should be before the END tag.

Reply:
> If the XMSS and LMS lines were manually generated

They are. The placement is correct (actually had to be corrected :) to avoid the "wipe out" by code-gen for "normal" PQ sig algs as pointed out by @dstebila